README.md

```diff
@@ -1,68 +1,119 @@
+
+## 2. Getting Access to Key Vault
+To make a long story short, you need a service principal to have access to key vault. The service principal serves as an application ID that is used during the authorization setup for access to other Azure resources. We'll use a Web App instance as our service principal. To do that, we create an App Service Plan, then a Web App instance, then make that our service principal. We can do all of this on Cloud Shell (click >_ on the top right hand corner in Portal to open). 
+
+1. Create App Service Plan : 
+   
+    ```az appservice plan create --name myServicePlanName --resource-group myResourceGroup --location westus```
+
+    There are locations other than westus. A json object will pop up when the command is done. 
+
+2. Create a Web App instance : (Note that the app name must be unique.)
+
+    ```az webapp create --name myUniqueAppName --plan myServicePlanName --resource-group myResourceGroup```
+
+3. Make the web app a service principal : 
+    
+    ```az ad sp create-for-rbac --name http://my-applications-url --skip-assignment```
+
+    Use your web app's url. To find that, click on the app in Portal, go to the 'Overview' tab and look for 'URL' on the top right portion of the page. It looks something like https://myUniqueAppName.azurewebsites.net. 
+
+    After the above comand runs, something like this is returned: 
+    ```
+        {
+           "appId": "11b855c6-43a5-415b-bd34-042a4509c179",
+           "displayName": "myUniqueAppName.azurewebsites.net",
+           "name": "https://myUniqueAppName.azurewebsites.net",
+           "password": "5ad92c90-9e0b-4bcb-a3ad-a121e9076af1",
+           "tenant": "72f998be-86f2-51ae-01af-2d5cd110db40"
+        }
+    ```
+
+    Later when you set environment variables, you'll need this info. The tenant will be saved as 'AZURE_TENANT_ID', appId as 'AZURE_CLIENT_ID', and password as 'AZURE_CLIENT_SECRET'. 
+
+    
+
+## 3. Setting Up The Project
 
 1. Clone the reponsitory
-```bash
-git clone https://github.com/jeffreymew/Flask-React-Postgres.git
-cd flask-react-postgres
-```
+   ```bash
+   git clone https://github.com/lilyjma/Flask-React-Postgres/tree/team_standup_app_cosmos
+   cd flask-react-postgres
+   ```
 
 2. Create and activate a virtual environment
 
-In Bash
-```bash
-python3 -m venv venv
-source venv/bin/activate
-```
-
-In Powershell
-```Powershell
-py -3 -m venv env
-env\scripts\activate
-```
+   In Bash
+   ```bash
+   python3 -m venv venv
+   source venv/bin/activate
+   ```
 
-In Powershell
-```Powershell
-py -3 -m venv env
-env\scripts\activate
-```
+   In Powershell
+   ```Powershell
+   py -3 -m venv env
+   env\scripts\activate
+   ```
 
 2. Install requirements.txt
-```bash
-pip install -r requirements.txt
-```
+   ```bash
+   pip install -r requirements.txt
+   ```
 
 3. Import the project folder into VS Code
-```bash
-code .
-```
+   ```bash
+   code .
+   ```
 
-## 2. Running The Code Locally
```

Comment:
Please containerize each part of the app so the user just has to start the container to run it.  Talk with me if you have questions.